### PR TITLE
docs: fix typos in some comments 

### PR DESCRIPTION
**Pull request by cuishuang**
## Change checklist

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: core/src/transport.rs [hunk 1]
```diff
@@ -311,7 +311,7 @@ pub enum TransportEvent<TUpgr, TErr> {
 
 impl<TUpgr, TErr> TransportEvent<TUpgr, TErr> {
     /// In case this [`TransportEvent`] is an upgrade, apply the given function
-    /// to the upgrade and produce another transport event based the the function's result.
+    /// to the upgrade and produce another transport event based the function's result.
```
Comment (jxs) [inline core/src/transport.rs]:
```suggestion
    /// to the upgrade and produce another transport event based on the function's result.
```

**Review by jxs — APPROVED** [review]
Thanks! LGTM can you just accept the suggestion? I don't seem to have permissions to commit it

**Comment by cuishuang** [comment]
Modified, please check again

**Review by jxs — APPROVED** [review]

**Comment by drHuangMHT** [comment]
@jxs Still cannot merge after labeled send-it 🙏

**Comment by jxs** [comment]
thanks for the ping @drHuangMHT it seems the permissions don't allow to merge:
>['Require a pull request before merging' branch protections are enabled on master, Mergify cannot check the pull request inplace.](https://github.com/libp2p/rust-libp2p/pull/5196/checks?check_run_id=22280610724)

can you change this @cuishuang ?
thanks

**Comment by cuishuang** [comment]
> 'Require a pull request before merging' branch protections are enabled on master, Mergify cannot check the pull request inplace.

Thanks!  Of course.

But how should I operate?

**Comment by jxs** [comment]
see [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

**Comment by cuishuang** [comment]
> see [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

image

Thank you. But the option I see in the bottom right corner is not available for selection

**Comment by jxs** [comment]
ah, it might be because you don't have those permissions over `rustaceanclub/libp2p` can you submit the PR from a fork owned by you?

**Comment by cuishuang** [comment]
> ah, it might be because you don't have those permissions over `rustaceanclub/libp2p` can you submit the PR from a fork owned by you?

This is the organization I created so there are absolute permissions.

I did something in the settings, please try again

**Comment by cuishuang** [comment]
image
something like this

**Comment by jxs** [comment]
@mergifyio refresh

**Comment by jxs** [comment]
@cuishuang seems to still have not be solved sorry

**Comment by cuishuang** [comment]
> @cuishuang seems to still have not be solved sorry

It's okay, sorry for wasting your precious time. I'll try to pull and submit it again~

**Comment by cuishuang** [comment]
> @cuishuang seems to still have not be solved sorry
Hi, I have resubmitted an MR, please review it again.
https://github.com/libp2p/rust-libp2p/pull/5255

**Pull request closed without merging**